bin/check-dns.rb

```diff
@@ -114,6 +121,10 @@ def check_against_regex(entries, regex)
   def run
     unknown 'No domain specified' if config[:domain].nil?
 
+    if config[:timeout].to_i < 5 || config[:timeout].to_i > 3600
```

Comment:
looking at the old version of this plugin (when it used `dig`) there every timeout < 5 was set to 5 - I might have copied that there. Not sure if it applies to the ruby library

I will just change it to use given value